src/Tests/README.md

```diff
@@ -0,0 +1,111 @@
+# Tests
+
+ONE-vscode's unittests are based on [mocha](https://mochajs.org/) and [@vscode/test-electron](https://github.com/microsoft/vscode-docs/blob/main/api/working-with-extensions/testing-extension.md).
+- use @vscode/test-electron because ONE-vscode uses vscode api
+  - use Extension Development Host which supports full access to the VS Code API
+- interface of mocha: TDD (BDD to test UX will be introduced in the future)
+- use `assert` of [chai](https://www.chaijs.com/api/assert/)
+
+## structure
+
+Tests are placed in `src/Tests`. The structure follows `src/`.
+```
+src/
+  ...
+  Project/
+    Builder.ts
+    Job.ts
+    ...
+  Utils/
+    Helpers.ts
+    ...
+```
```

Comment:
I am not sure the purpose of showing L12 to L22 here. 
Can you add some caption for L12 to L22 and another caption for L23 to L42?